src/app/public/modules/repeater/repeater.component.ts

```diff
@@ -67,17 +75,22 @@ export class SkyRepeaterComponent implements AfterContentInit, OnChanges, OnDest
     return this._expandMode || 'none';
   }
 
+  @Output()
+  public activeIndexChange = new EventEmitter();
+
   @ContentChildren(SkyRepeaterItemComponent)
   public items: QueryList<SkyRepeaterItemComponent>;
 
   private ngUnsubscribe = new Subject<void>();
 
+  private _activeIndex: number;
+
   private _reorderable = false;
 
   private _expandMode = 'none';
 
   constructor(
-    private repeaterService: SkyRepeaterService,
+    public repeaterService: SkyRepeaterService,
```

Comment:
Curious why this needs to be public?

Reply:
So the template could access it. On further investigation, a new public method for getting that data would do just fine. Reverted this back to `private`.